src/main/java/io/github/thebusybiscuit/slimefun4/integrations/OrebfuscatorIntegration.java

```diff
@@ -61,4 +64,14 @@ public void onExplosiveToolBreakBlocks(ExplosiveToolBreakBlocksEvent event) {
     public void onReactorExplode(ReactorExplodeEvent event) {
         this.service.deobfuscate(Arrays.asList(event.getLocation().getBlock()));
     }
+
+    @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
+    public void onGoldPanUse(PlayerRightClickEvent event) {
+        if (event.getSlimefunItem().isPresent() && event.getClickedBlock().isPresent()) {
+            String id = event.getSlimefunItem().get().getId();
+            if (id.equals(SlimefunItems.GOLD_PAN.getItemId()) || id.equals(SlimefunItems.NETHER_GOLD_PAN.getItemId())) {
+                this.service.deobfuscate(List.of(event.getClickedBlock().get()));
+            }
+        }
+    }
```

Comment:
I would replace this with an instanceof GoldPan check, so `event.getSlimefunItem().get() instanceof GoldPan`

Reply:
Smart, changed now